fabtests/functional/multi_recv.c

```diff
@@ -453,7 +266,8 @@ int main(int argc, char **argv)
 	int op, ret;
 
 	opts = INIT_OPTS;
-	opts.options |= FT_OPT_SIZE | FT_OPT_SKIP_MSG_ALLOC;
+	opts.options |= FT_OPT_SIZE | FT_OPT_SKIP_MSG_ALLOC | FT_OPT_OOB_SYNC |
+			FT_OPT_OOB_ADDR_EXCH | FT_OPT_SKIP_MSG_ALLOC;
```

Comment:
SKIP_MSG_ALLOC is or'ed in twice.

Reply:
Gotta or it in twice because it might get compressed out after the verbs compression happens, you know?